### DISCUSSION TITLE
Is it possible to achieve the minimum load on the cpu?

### DISCUSSION THREAD

**Discussion by tequilaonelove** [AzuraCast/AzuraCast#4864] [Q&A]
How to minimize the load on the server from Azuracast if Azuracast is used only as a stream repeater at the connection point?
Periodically I see the load on the server 35-40% in htop from `php bin / console sync: run nowplaying`

![35355](https://user-images.githubusercontent.com/27417279/145689247-b75a4d73-5cb9-4609-8801-af9746d16066.png)

In theory, from Azuracast, I need such a function as if it were a simple lightweight mount in icecast2, without any synchronization and heavy additional operations, etc. Is it possible to pacify/calm my attitude a little?

**Comment by tequilaonelove** [comment]
?

**Reply by spazztula** [reply]
Hi have you figured out how to do this, as I see you are running on a single CPU machine with 2GB RAM and some swap similar to my setup which is 1CPU 2GB RAM and 4.5BG swapfile. After some latest updates  my instance has had same higher CPU usage. Currently on AzuraCast • v0.20.1 Stable
![image](https://github.com/user-attachments/assets/75309b69-617b-47bf-976d-dcc5fe8a0946)


Possible similar issues on [https://github.com/AzuraCast/AzuraCast/issues/7313](url) and [https://github.com/AzuraCast/AzuraCast/issues/7207](url)
[https://github.com/AzuraCast/AzuraCast/issues/7173](url)
